tests/lwip/Makefile

```diff
@@ -1,10 +1,25 @@
 include ../Makefile.tests_common
 
+LWIP_IPV4 ?= 0
```

Comment:
I wish we had a way for Murdock to exercise both cases, but afaik that's not possible.

Reply:
Maybe, there is a way to extend the make system by a variable that define environment variants in a Makefile or any other additional file that is then used in compile tools to compile the different variants.